eng/pipelines/templates/pipeline.yml

```
@@ -45,6 +45,7 @@ resources:
 
 variables:
   DOTNET_NOLOGO: 1
+  NUGET_LOCTYPE: Full
```

Should we respect casing in variable name?

```suggestion
  NuGetLocType: Full
```